code/modules/vehicles/mecha/_mecha.dm

```diff
@@ -670,9 +670,9 @@
 
 	//if we're not facing the way we're going rotate us
 	if(dir != direction && !strafe || forcerotate || keyheld)
-		setDir(direction)
-		if(turnsound && dir != direction)
+		if(dir != direction && !(mecha_flags & QUIET_STEPS) && !step_silent)
```

Comment:
```suggestion
		if(dir != direction && !(mecha_flags & QUIET_TURNS) && !step_silent)
```
also why does it check step_silent? wouldn't turning make noise regardless of gravity?